src/stores/spaces/SpaceStore.ts

```diff
@@ -854,6 +854,13 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
                 break;
             }
 
+            case "view_home_page":
+                if (this.enabledMetaSpaces.includes(MetaSpace.Home)) {
+                    this.setActiveSpace(MetaSpace.Home, false);
+                    window.localStorage.setItem(getSpaceContextKey(this.activeSpace), "");
```

Comment:
Should the `localStorage` call be inside the `setActiveSpace` method? It sounds like we will otherwise duplicate that code everywhere?

Reply:
There are only two places which set this localStorage field, vs many many calls to `setActiveSpace` - avoids many redundant localStorage writes